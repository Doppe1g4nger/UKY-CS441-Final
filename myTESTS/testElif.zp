 int main ()
{
    int sum;
    int i;
    sum = getnum();
    if (sum < 35)
    {
        puts("Less than 35\n");
    }
    else if (sum < 50)
    {
        puts("Less than 50\n");
    }
    return 0;
}
