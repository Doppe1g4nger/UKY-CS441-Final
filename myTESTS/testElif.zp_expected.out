Less than 35
Less than 50
